GUI.py

```diff
@@ -586,22 +541,19 @@ def get_files_list(path_to_dir):
     return np.array(f)
 
 
-def refection_coef_read(path_to_file):
+def refection_coef_read(path_to_file: str) -> (np.ndarray, np.ndarray):
     if os.path.exists(path_to_file):
         n_lam = pd.read_csv(
-            path_to_file, header=None, delimiter=",", encoding="utf-8"
-        ).rename(columns={0: "x", 1: "y"})
-    wvlngh = n_lam["x"].values
-    n_coef = n_lam["y"].values
-
-    return wvlngh, n_coef
+            path_to_file, header=0, delimiter=",", encoding="utf-8", names=["x", "y"]
+        )
+    return n_lam["x"].values, n_lam["y"].values
 
 
-def thickness(wavelength, period, n):
-    return np.round(wavelength * wavelength / (2.0 * n * period * 10000.0), 2)
+def thickness(wavelength: float, period: float, n: float) -> float:
+    return np.round(wavelength ** 2 / (2.0 * n * period * 10000.0), 2)
 
 
-def fourier_analysis(x, y):
+def fourier_analysis(x: np.ndarray, y: np.ndarray) -> (np.ndarray, np.ndarray, float, float):
```

Comment:
change to `-> Tuple[np.ndarray, np.ndarray, float, float]`

and above: `from typing import Tuple`

Reply:
+ Docstring